waiting.html

```diff
@@ -1,61 +1,32 @@
+{{define "waiting"}}
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
-  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0"/>
-  <title>Starter Template - Materialize</title>
-
-  <!-- CSS  -->
-  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-  <link href="assets/css/materialize.css" type="text/css" rel="stylesheet" media="screen,projection"/>
-  <link href="assets/css/style.css" type="text/css" rel="stylesheet" media="screen,projection"/>
-</head>
+{{template "header" . }}
 <body>
-  <nav class="light-blue lighten-1" role="navigation">
-    <div class="nav-wrapper container"><a id="logo-container" href="#" class="brand-logo">Trashies</a>
-      <a href="index.html" data-activates="nav-mobile" class="button-collapse"><i class="material-icons">menu</i></a>
-    </div>
-  </nav>
-  <div class="section no-pad-bot" id="index-banner">
+{{template "navbar" . }}
+<div class="section no-pad-bot" id="index-banner">
     <div class="container">
-      <br><br>
-      <h1 class="header center orange-text">Please Wait...</h1>
-      <div class="row center">
-        <h5 class="header col s12 light">We'll get moving shortly, unless you're here way early then I don't know what you want from me.</h5>
-          <div>
-            <img src="images/hot-dog-boi.gif"/>
-          </div>
-      <br><br>
-    </div>
-  </div>
-
-  <div class="container">
-    <div class="section">
+        <br><br>
+        <h1 class="header center orange-text">Please Wait...</h1>
+        <div class="row center">
+            <h5 class="header col s12 light">We'll get moving shortly, unless you're here way early then I don't know
+                what you want from me.</h5>
+            <div>
+                <img src="images/hot-dog-boi.gif"/>
+            </div>
+            <br><br>
+        </div>
     </div>
-    <br><br>
-  </div>
 
-  <footer class="page-footer orange">
     <div class="container">
-      <div class="row">
-        <div class="col l6 s12">
-          <h5 class="white-text">Company Bio</h5>
-          <p class="grey-text text-lighten-4">We are become trash</p>
+        <div class="section">
         </div>
-      </div>
-    </div>
-    <div class="footer-copyright">
-      <div class="container">
-      Made by <a class="orange-text text-lighten-3" href="https://www.youtube.com/watch?v=SW-BU6keEUw">Idiots</a>
-      </div>
+        <br><br>
     </div>
-  </footer>
-
 
-  <!--  Scripts-->
-  <script src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
-  <script src="js/materialize.js"></script>
-  <script src="js/init.js"></script>
+{{template "footer" . }}
+{{template "jsimports" . }}
 
-  </body>
+</body>
 </html>
+{{ end }}
```

Comment:
newline here

Reply:
weird, its there on my local copy but not on the pushed version.